src/main/java/frc/robot/UnitConversions.java

```diff
@@ -1,9 +1,27 @@
 package frc.robot;
 
 public class UnitConversions {
+    public static class Indexer {
+        public static double gateWheelEncoderTicksToDegrees (double encoderTicks) {
+            return (encoderTicks / 4096 * Constants.Indexer.GATE_WHEEL_GEAR_RATIO) * 360;
+        }
+
+        public static double gateWheelDegreesToEncoderTicks (double degrees) {
+            return degrees / gateWheelEncoderTicksToDegrees(1);
+        }
+
+        public static double gateWheelEncoderTicksPer100msToRPM (double encoderTicksPer100ms) {
+            return (gateWheelEncoderTicksToDegrees(encoderTicksPer100ms) / 360.0) * 10.0 * 60.0;
+        }
+
+        public static double gateWheelRPMToEncoderTicksPer100ms (double RPM) {
+            return RPM / gateWheelEncoderTicksPer100msToRPM(1);
+        }
+    }
```

Comment:
Double check if all of these functions are required to be public, some can be private if we aren't expecting to use them anywhere.

Also I see a lot of math happening, might be good to comment exactly what the values mean and what the formula is doing so other can understand.

Lastly, you may be able to simplify this if you are expecting to call some or all of these functions often, removing some of the math and resolving them down as much as possible will remove unneeded calculations. (This doesn't matter if it's only run every once in a while but if we are running these functions multiple times per update of the system that can be expensive)